src/scene/shader-lib/programs/lit-shader.js

```diff
@@ -474,7 +474,8 @@ class LitShader {
         code += this.frontendDecl;
         code += this.frontendCode;
 
-        const usePackedDepth = (!device.supportsDepthShadow) && (shadowType === SHADOW_PCF1 || shadowType === SHADOW_PCF3 || shadowType === SHADOW_PCSS);
+        const usePackedDepth = (!device.supportsDepthShadow) && (shadowType === SHADOW_PCF1 || shadowType === SHADOW_PCF3 || shadowType === SHADOW_PCSS) ||
+            (lightType === LIGHTTYPE_OMNI && !options.clusteredLightingEnabled);
```

Comment:
```suggestion
        const usePackedDepth =
            (!device.supportsDepthShadow) &&
            (shadowType === SHADOW_PCF1 || shadowType === SHADOW_PCF3 || shadowType === SHADOW_PCSS) ||
            (lightType === LIGHTTYPE_OMNI && !options.clusteredLightingEnabled);

```

Reply:
Made it more clear that the supportsDepthShadow test and the PCF1/3 and PCSS methods are dependent on it by adding an extra set of parenthesis.